BChat/Conversations/ConversationVC+Interaction.swift

```diff
@@ -465,24 +465,30 @@ extension ConversationVC : InputViewDelegate, MessageCellDelegate, ContextMenuAc
                                                 inputTextView.textColor = Colors.accent
                                                 let value = ["bdxAmount": newText]
                                                 NotificationCenter.default.post(name: .bdxAmountPassingSliderViewNotification, object: value)
+                                                CustomSlideViewBool.isFromExpandAttachment = true
```

Comment:
Change the var name (  CustomSlideViewBool)

Reply:
customSlideView name changed.